src/test/java/nl/tudelft/ti2206/group9/level/entity/AbstractEntityTest.java

```diff
@@ -22,12 +24,16 @@
 
     private transient AbstractEntity entity;
 
+    private CrashMap crashMap;
+
     @Before
     public void setUp() throws Exception {
-        first = mock(AbstractEntity.class);
-        second = mock(AbstractEntity.class);
+        first = new AbstractEntityTest.TestEntity(Point3D.ZERO, Point3D.UNITCUBE);
+        second = new AbstractEntityTest.TestEntity(Point3D.ZERO, Point3D.UNITCUBE);
```

Comment:
You made these two entities non-mocked because your test didn't want to work. Now the test does work, can these two be mocked again?

<review, testing